app/config/keymaps.js

```diff
@@ -1,4 +1,5 @@
 const {readFileSync} = require('fs');
+const normalize = require('../utils/keymaps/normalize');
 const {defaultPlatformKeyPath} = require('./paths');
```

Comment:
@chabou you can make a PR for this 😂  `defaultPlatformKeyPath` should maybe be : `defaultFormKeyPath`